app/presenters/decidim/decidim_awesome/paper_trail_role_presenter.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Decidim
+  module DecidimAwesome
+    class PaperTrailRolePresenter < Decidim::Log::BasePresenter
+      include TranslatableAttributes
+
+      delegate :created_at, to: :entry
+
+      attr_reader :entry
+
+      def initialize(entry)
+        @entry = entry
+      end
+
+      def role
+        role = destroy_action(entry)&.reify&.role || entry.item&.role
+        role == "admin" ? "administrator" : role
```

Comment:
We need to extract the i18n version of the role
```suggestion
        role == t("roles.#{role}", scope: "decidim.decidim_awesome.admin_accountability", default: role)
```
We know about this roles currently:
```yaml
roles:
            admin: Administrator
            collaborator: Collaborator
            moderator: Moderator
            valuator: Valuator
```